Extensions/auto_tagger.js

```diff
@@ -269,7 +269,12 @@ XKit.extensions.auto_tagger = new Object({
 
 		if (XKit.extensions.auto_tagger.preferences.tag_person.value && obj.owner !== "") {
 			if (XKit.extensions.auto_tagger.preferences.tag_person_replace_hyphens.value) {
-				to_return = this.mreturn_add(to_return, obj.owner.replace(/-/g, ' '));
+				if (obj.owner !== undefined) {
```

Comment:
This only solves one very specific case due to the use of !== undefined here, but I'll merge this as a temporary fix because the actual fix looks like it will end up involving a major API change.